specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-10-01-preview/ThreatIntelligence.json

```diff
@@ -522,9 +612,138 @@
         "$ref": "#/definitions/ThreatIntelligenceAppendTags"
       },
       "x-ms-parameter-location": "method"
+    },
+    "ThreatIntelligenceQueryByCondition": {
+      "description": "Filtering criteria for querying threat intelligence indicators by condition.",
+      "in": "body",
+      "name": "filterCriteria",
+      "required": true,
+      "schema": {
+        "$ref": "#/definitions/ThreatIntelligenceQueryByCondition"
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "ThreatIntelligenceCountByCondition": {
+      "description": "Filtering criteria for counting threat intelligence indicators by condition.",
+      "in": "body",
+      "name": "filterCriteria",
+      "required": true,
+      "schema": {
+        "$ref": "#/definitions/ThreatIntelligenceCountByCondition"
+      },
+      "x-ms-parameter-location": "method"
     }
   },
   "definitions": {
+    "Connective": {
+      "description": "Connective used to join the conditions or clauses together.",
+      "enum": [
+        "and",
+        "or"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "Connective",
+        "values": [
+          {
+            "description": "Conditions/clauses should be joined together using logical AND operator.",
+            "value": "and"
+          },
+          {
+            "description": "Conditions/clauses should be joined together using logical OR operator.",
+            "value": "or"
+          }
+        ]
+      }
+    },
+    "Clause": {
+      "description": "Represents a clauses in a condition.",
+      "type": "object",
+      "properties": {
+        "clauseConnective": {
+          "description": "Connective used to logically join the values on this clause",
+          "$ref": "#/definitions/Connective"
+        },
+        "field": {
```

Comment:
>field [](http://example.com/codeflow?start=9&length=5)

Could this be an enum? Also, it looks like some other parts of the API call this column, not field. Recommend trying to be consistent here (unless field != column, which I'd find also pretty confusing).

Reply:
It is not an enum. Also yes, technically field == column but we are choosing to go with 'field' for these new endpoints and any other future endpoints.